tiledb/sm/array_schema/array_schema.cc

```diff
@@ -339,6 +339,11 @@ Status ArraySchema::check() const {
     }
   }
 
+  if (array_type_ == ArrayType::SPARSE && capacity_ == 0)
+    return LOG_STATUS(
```

Comment:
- please add braces around if body (error prone)
- and `throw` here (LOG_STATUS returns the object, and all `Status_` are interconvertible to exceptions)

Reply:
OK. I was doing what other placed in the file did, more gradual migration I suppose?